RequestService/RequestService.Handlers/PostNewRequestForHelpHandler.cs

```diff
@@ -42,7 +42,7 @@ private void CopyRequestorAsRecipient(PostNewRequestForHelpRequest request)
         {
             request.HelpRequest.Requestor.Address.Postcode = HelpMyStreet.Utils.Utils.PostcodeFormatter.FormatPostcode(request.HelpRequest.Requestor.Address.Postcode);
 
-            if (request.HelpRequest.ForRequestor)
+            if (request.HelpRequest.RequestorType != RequestorType.OnBehalf)
```

Comment:
Should this not be == RequestorType.Myself ?

Reply:
if requester type is organisation then we also need to add the requestor as as the recipient